specifications/raml/api.raml

```diff
@@ -227,17 +145,8 @@ traits:
           200:
             body:
               application/json:
-                schema: |
-                  {
-                    "$schema": "http://json-schema.org/draft-03/schema",
-                    "type": "object",
-                    "properties": {
-                      "tracks": {
-                        "type": "array",
-                        "items": {"$ref": "track"}
-                      }
-                    }
-                  }
+                type: Track[]
```

Comment:
Yes, it would have been better to just return an array of tracks, but we decided to return every reply in a wrapped object. I like this syntax, but if would be good if we could return the right structure.